assertRows failed in PostgreSQL DML integration test

## Bug Report

### Which version of ShardingSphere did you use?
master branch, 5.0.0-RC1-SNAPSHOT

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-JDBC

### Expected behavior
Test case pass

### Actual behavior
Exception thrown:
```
[main] ShardingSphere-SQL - Actual SQL: db_1 ::: INSERT INTO t_order (order_id, user_id, status) VALUES (1, 1, 'insert')
[main] o.a.s.dbtest.engine.dml.BaseDMLIT - [ERROR] SQL::INSERT INTO t_order (order_id, user_id, status) VALUES (?, ?, ?), Parameter::1:int, 1:int, insert:String, Expect::insert_for_order_1.xml

java.lang.AssertionError: 
Expected: is "1"
     but: was "1100"
Expected :1
Actual   :1100
<Click to see difference>

	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.apache.shardingsphere.dbtest.engine.dml.BaseDMLIT.assertRows(BaseDMLIT.java:148)
	at org.apache.shardingsphere.dbtest.engine.dml.BaseDMLIT.assertDataSet(BaseDMLIT.java:114)
	at org.apache.shardingsphere.dbtest.engine.dml.BaseDMLIT.assertDataSet(BaseDMLIT.java:102)
	at org.apache.shardingsphere.dbtest.engine.dml.GeneralDMLIT.assertExecuteUpdate(GeneralDMLIT.java:75)
```

### Reason analyze (If you can)
PostgreSQL query won't order records by primary key ascending by default. e.g.
```
db_1=# select * from t_order;
 order_id | user_id | status
----------+---------+--------
     1100 |      11 | init
     1101 |      11 | init
     2100 |      21 | init
     2101 |      21 | init
        1 |       1 | insert
```

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
1, env.properties:
rule.types=db
databases=PostgreSQL
postgresql.host=127.0.0.1
postgresql.password=test

2, Run `GeneralDMLIT` in IDEA

### Example codes for reproduce this issue (such as a github link).

--- a/shardingsphere-integration-test/shardingsphere-test-suite/src/test/java/org/apache/shardingsphere/dbtest/engine/dml/BaseDMLIT.java
+++ b/shardingsphere-integration-test/shardingsphere-test-suite/src/test/java/org/apache/shardingsphere/dbtest/engine/dml/BaseDMLIT.java
@@ -17,6 +17,7 @@
 
 package org.apache.shardingsphere.dbtest.engine.dml;
 
+import java.sql.Statement;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.shardingsphere.dbtest.cases.assertion.dml.DMLIntegrateTestCaseAssertion;
 import org.apache.shardingsphere.dbtest.cases.assertion.root.SQLCaseType;
@@ -27,6 +28,7 @@ import org.apache.shardingsphere.dbtest.cases.dataset.row.DataSetRow;
 import org.apache.shardingsphere.dbtest.engine.SingleIT;
 import org.apache.shardingsphere.dbtest.env.EnvironmentPath;
 import org.apache.shardingsphere.dbtest.env.dataset.DataSetEnvironmentManager;
+import org.apache.shardingsphere.infra.database.type.dialect.PostgreSQLDatabaseType;
 import org.apache.shardingsphere.sharding.algorithm.sharding.inline.InlineExpressionParser;
 import org.apache.shardingsphere.infra.database.type.DatabaseType;
 import org.apache.shardingsphere.infra.datanode.DataNode;
@@ -94,8 +96,17 @@ public abstract class BaseDMLIT extends SingleIT {
             DataSetMetadata expectedDataSetMetadata = expected.getMetadataList().get(0);
             for (String each : new InlineExpressionParser(expectedDataSetMetadata.getDataNodes()).splitAndEvaluate()) {
                 DataNode dataNode = new DataNode(each);
+                String sql;
+                if (getDatabaseType() instanceof PostgreSQLDatabaseType) {
+                    try (Connection connection = getDataSourceMap().get(dataNode.getDataSourceName()).getConnection()) {
+                        String primaryKeyColumnName = getPostgreSQLTablePrimaryKeyColumnName(connection, dataNode.getTableName());
+                        sql = String.format("SELECT * FROM %s ORDER BY %s ASC", dataNode.getTableName(), primaryKeyColumnName);
+                    }
+                } else {
+                    sql = String.format("SELECT * FROM %s", dataNode.getTableName());
+                }
                 try (Connection connection = getDataSourceMap().get(dataNode.getDataSourceName()).getConnection();
-                     PreparedStatement preparedStatement = connection.prepareStatement(String.format("SELECT * FROM %s", dataNode.getTableName()))) {
+                     PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
                     assertDataSet(preparedStatement, expected.findRows(dataNode), expectedDataSetMetadata);
                 }
             }
@@ -109,6 +120,18 @@ public abstract class BaseDMLIT extends SingleIT {
         try (ResultSet actualResultSet = actualPreparedStatement.executeQuery()) {
             assertMetaData(actualResultSet.getMetaData(), expectedDataSetMetadata.getColumns());
             assertRows(actualResultSet, expectedDataSetRows);
+        }
+    }
+    
+    private String getPostgreSQLTablePrimaryKeyColumnName(final Connection connection, final String tableName) throws SQLException {
+        String sql = "SELECT a.attname, format_type(a.atttypid, a.atttypmod) AS data_type FROM pg_index i JOIN pg_attribute a "
+            + "ON a.attrelid = i.indrelid AND a.attnum = ANY(i.indkey) WHERE i.indrelid = '"
+            + tableName + "'::regclass AND i.indisprimary";
+        try (Statement statement = connection.createStatement(); ResultSet resultSet = statement.executeQuery(sql)) {
+            if (!resultSet.next()) {
+                throw new SQLException("can not get primary key of " + tableName);
+            }
+            return resultSet.getString("attname");
         }
     }
     

--- a/shardingsphere-integration-test/shardingsphere-test-suite/src/test/java/org/apache/shardingsphere/dbtest/env/authority/AuthorityEnvironmentManager.java
+++ b/shardingsphere-integration-test/shardingsphere-test-suite/src/test/java/org/apache/shardingsphere/dbtest/env/authority/AuthorityEnvironmentManager.java
@@ -83,10 +83,7 @@ public final class AuthorityEnvironmentManager {
         try (Connection connection = dataSource.getConnection()) {
             for (String each : sqls) {
                 try (Statement statement = connection.createStatement()) {
-                    boolean executeRet = statement.execute(each);
-                    if (!executeRet) {
-                        System.err.println("execute '" + each + "' failed");
-                    }
+                    statement.execute(each);
                 } catch (final SQLException ex) {
                     System.err.println("execute '" + each + "' failed, ex.msg=" + ex.getMessage());
                 }

